README.rst

```diff
@@ -72,6 +70,41 @@ To enable the ``unregister`` and ``register`` endpoints you can add the followin
 Usage
 -----
 
+Supported Services
+~~~~~~~~~~~~~~~~~~
+
+Following services are provided with configuration required for them:
+
+1. Apple Push Notification Service(APNs):
+
+.. code:: python
+
+    DJANGO_PUSH_NOTIFICATIONS = {
+        'SERVICE': 'push_notifications.services.apns.APNSPushService',
+        'APNS_CONF': {
+            'APNS_DEBUG': False,  # Determines whether to use Apple Production Gateway or Sandbox servers
+            'APNS_CERTIFICATE': None,  # Full path to APNS certificate file. Certificates should be without passphrase.
+            'APNS_HOST': 'gateway.push.apple.com',
+            'APNS_FEEDBACK_HOST': 'feedback.push.apple.com',
+            'APNS_PORT': 2195,´
+            'APNS_FEEDBACK_PORT': 2196,
+            'APNS_ERROR_TIMEOUT': None,  # Custom timeout for ssl connection
+            'APNS_MAX_NOTIFICATION_SIZE': 2048,
```

Comment:
Only certificate is necessary, others are optional. I didn't make it clear here.